.github/ISSUE_TEMPLATE/new_program_type.md

```diff
@@ -0,0 +1,19 @@
+---
+name: New program type
+about: Request supporting additonal interface
+title: ''
+labels: 'run,configs'
+assignees: ''
+
+---
+
+*Please note that we can support only deterministic programs.*
+
+- Proposed name:
+- Program role: generator / validator / judge
+
+**Describe the interface**
+Add accurate description of the program interface, i.e. command line arguments, environment variables, used files etc.
```

Comment:
The heading should probably be followed by a paragraph break or a colon. Unless we expect the users to remove this heading, it should also probably make sense once filled in.

```suggestion
**Interface description:**
Add an accurate description of the program interface, i.e. command line arguments, environment variables, used files, etc.
```